README.md

```diff
@@ -1 +1,40 @@
-# CeaselessAlexa
\ No newline at end of file
+# Ceaseless Alexa Skill
+
+The Alexa skill is comprised of two components: (1) the server code and (2) the Alexa skill configuration.
+
+## Server Code
+
+The server code for the skill is written in C# and runs on .NET Core 1.0. The server code is then deployed to AWS Lambda and ran as a serverless function.
+
+[![Build status](https://ci.appveyor.com/api/projects/status/ueprnsv2ade3l0b7/branch/master?svg=true)](https://ci.appveyor.com/project/adriangodong/ceaselessalexa/branch/master)
+
+### Developer Workflow for Server Code
+
+0. Download and install [.NET Core SDK](https://www.microsoft.com/net/download/core)
+1. Build and test using the following command:
+
+       dotnet restore && dotnet build && dotnet test Ceaseless.Tests/Ceaseless.Tests.csproj
+
+2. To publish code to AWS Lambda, run the following command:
+
+       cd Ceaseless
+       dotnet lambda deploy-function
```

Comment:
Made it all the way to this step. Getting this error:
No executable found matching command "dotnet-lambda"